src/components/Apps/AppNavButtons.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+
+import { getHomeApp } from 'lib/reducers'
+
+import Icon from 'cozy-ui/react/Icon'
+import homeIcon from 'assets/icons/icon-cozy-home.svg'
+
+class AppNavButton extends Component {
+  render () {
+    const { homeApp, busy, onClick, appName, appNamePrefix, iconPath, opened } = this.props
```

Comment:
Global remark:

I don't feel realy comfortable having a ambiguous var name here, as `home` means the _Home page_, but also it the slug of our `Home app`. So it feels we're using directly a reference to the Home app, as we did in the past with Collect for example, and it's a code smell.

But I also agree that `home` is also the best word to describe this variable. What alternative do we have ? `default` ? `entry` ? `main` ?